### ISSUE TITLE
Change font for source and target languages

### ISSUE THREAD

**Issue by mondele** [friendsofagape/autographa-lite#114]
It should be possible to change the display (and editing) font for source and target language panes.

**Comment by joelthe1** [comment]
Could you explain your use-case a bit more as to why this would be helpful in your context?

**Comment by mondele** [comment]
Yes - we're doing translation in a language from one of your neighboring countries. The typical Arabic font rendering (for example, Noto) doesn't look quite right to the native speakers, and they have trouble reading it. Installation of a different font resolves that problem, but Autographa doesn't support it.

**Comment by joelthe1** [comment]
Thanks for that. We will put this on the list. To give you some more context, the team is planning on integrating Graphite font rendering engine that would allow such text to show-up properly. This is dependent on an external team doing this for us and will keep you posted on the progress. Ideally, this should solve your problem.

**Comment by mondele** [comment]
I agree that graphite support will support more fonts, but right now there is just no option to select the display font (although the font size adjustment is beautifully intuitive).

John Wood
john_wood@wycliffeassociates.org<mailto:john_wood@wycliffeassociates.org>
[cid:dd59814d8c593e539335dbd1dc33bb75b36fa0f3.camel@wycliffeassociates.org]MAST Help Desk Supervisor
@jwood on #slack
archistratus on [cid:bfa9fcaedcdc412b12590c249746c3b6cb694bf6.camel@wycliffeassociates.org]
https://t.me/toubabou - Telegram
Toubabou on Keybase
Messages may be signed with PGP
Small attachments are probably the logos and the signature


On Aug 20, 2019, at 2:45 AM, Joel Mathew <notifications@github.com<mailto:notifications@github.com>> wrote:


Thanks for that. We will put this on the list. To give you some more context, the team is planning on integrating Graphite font rendering engine that would allow such text to show-up properly. This is dependent on an external team doing this for us and will keep you posted on the progress. Ideally, this should solve your problem.
